.gitignore

```diff
@@ -3,3 +3,4 @@ pkg
 *.gem
 spec/sandbox
 Gemfile.lock
+.idea
```

Comment:
Is that specific to `huginn_agent`? If not I think it should be in`~/.gitignore` 😄

Reply:
No, it's for all JetBrains IDEs (like RubyMine). I can remove it, though.

Reply:
I see, yes please, in my opinion we should not have IDE/editor specific files in here.

Reply:
Oops, meant to push the removal!